can_tools/scrapers/official/federal/CDC/cdc_county_cases_deaths.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+import pandas as pd
+from can_tools.scrapers.official.base import ETagCacheMixin, FederalDashboard
+from can_tools.scrapers import variables
+from multiprocessing import get_context
+import requests
+
+from can_tools.scrapers.base import ALL_STATES_PLUS_DC
+
+
+class CDCCountyCasesDeaths(FederalDashboard, ETagCacheMixin):
+    has_location = True
+    location_type = "county"
+    source = "https://data.cdc.gov/Case-Surveillance/Weekly-United-States-COVID-19-Cases-and-Deaths-by-/pwn4-m3yp"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc"
+    fetch_url = "https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=integrated_county_timeseries_fips_{county_fips}_external"
```

Comment:
CDC surfaces raw weekly cases on their [county view](https://covid.cdc.gov/covid-data-tracker/#county-view?list_select_state=Florida&data-type=CommunityLevels&list_select_county=12086) but I couldn't find this in any official dataset, so I pulled from the API that powers this page. 

An alternative would be to recalculate weekly cases from their [metrics dataset](https://data.cdc.gov/Public-Health-Surveillance/Weekly-COVID-19-County-Level-of-Community-Transmis/jgk8-6dpn/data).

<img width="604" alt="image" src="https://user-images.githubusercontent.com/55333380/220807200-548ff7d2-4ab8-411d-afac-6d660575d920.png">